CHANGES.md

```diff
@@ -1,4 +1,6 @@
 # UNRELEASED
+- [NEW] The underlying Cloudant client library is now EOL. Refactored implementation and test suite to
```

Comment:
Nit, it is new, but it's also an upgrade and I think we can keep it short and sweet.
```suggestion
- [UPGRADED] Refactored to use the new `cloudant-java-sdk` library.
```